docs/DEPLOYMENT.md

```diff
@@ -27,59 +27,78 @@ Setting up staging environment on AWS is pretty straightforward.
    2. Name it `plio-staging-vpc`.
    3. In IPv4 CIDR block, enter `10.0.0.0/28`.
    4. Click on create button. You will see the new VPC under the list of VPCs.
-3. Create a new Elastic IP by going to EC2 dashboard and navigating to Elastic IP section.
+3. Set up database through RDS.
+   1. Click on create database.
+   2. Use "Standard create" in database creation method.
+   3. Use Postgres12 in the DB engine.
+   4. Under templates, go with "Production".
+   5. Name the DB instance identifier as `plio`.
+   6. Set the master user name and password.
+   7. For DB instance, we can go with Burstable classes to get started with. Select the one having vCPUs & RAM as per your needs.
+   8. Set the alloted storage as per your needs.
+   9. For Availability and Durability, use multi-AZ deployment if you need. Take a note that this may almost double your monthly expense.
+   10. Under VPC settings, select the VPC created in previous step.
+   11. In case you prefer to connect to your database directly from your machine, set the public access to "Yes".
+   12. Under additional configuration, set initial database name as `plio`. This will create an empty database.
+   13. Create the RDS instance by clicking on "Create Database" button.
+   14. You will see the new RDS instance in the list of RDS instances. Click on your DB instance to see the connectivity settings.
+   15. To connect to RDS from command line, run the following command (this will work if you've set public access to yes):
+   ```sh
+   psql -h your-db-instance-endpoint.aws-region.rds.amazonaws.com -p 5432 -U master_username
```

Comment:
we should add a line saying that if one wants to create another DB on the same instance, they should connect to their instance using this line and then use the CREATE DATABASE and other commands for creating that database with a new user for example.